src/topp/DecoyDatabase.cpp

```diff
@@ -96,6 +108,15 @@ class TOPPDecoyDatabase :
     setValidStrings_("enzyme", all_enzymes);
 
     registerSubsection_("Decoy", "Decoy parameters section");
+
+    // New options for neighbor peptide search
+    registerTOPPSubsection_("Neighbor_Search", "Parameters for neighbor peptide search");
+    registerInputFile_("Neighbor_Search:neighbor_in", "<file>","", "Input FASTA file for neighbor peptide search", false);
```

Comment:
not clear what this is.
Is it the relevant peptides or the large DB from which to pick neighbor peptides?